base/doc/ltnews33.tex

```diff
@@ -696,6 +696,14 @@ \subsection{\pkg{longtable}: General bug fix update}
 
 
 
+\subsection{\pkg{bm}: Better support for commands with optional arguments}
+Some uses of optional arguments that were supported by \cs{bm} stopped
+being supported (in 2004) when \cs{kernel@ifnextchar} was used
+internally by the format instead of \cs{@ifnextchar} This update
```

Comment:
period missing

Reply:
@FrankMittelbach  I added the . I'll have to think about 71 hold off on the PR for a bit (I'll flip it to draft)